### PR TITLE
Add change control to the request payload tables

### PR DESCRIPTION
**Pull request by alekhyamoz**
Relates to https://mozilla-hub.atlassian.net/browse/RS-698

TL;DR The request payload tables should be changed without the necessary approvals.
The primary owners are Wil Stuckey and Rebecca BurWei and one of their approval is required to make changes to the ETL logic.
Tagging both of them as reviewers in this PR for awareness

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/contextual_services_derived/request_payload_suggest_v2/metadata.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@ description: |-
 owners:
   - skahmann@mozilla.com
   - akommasani@mozilla.com
+  - mozilla/request_payload_reviewers
 labels:
   incremental: true
   schedule: daily
```
Comment (lucia-vargas-a) [inline sql/moz-fx-data-shared-prod/contextual_services_derived/request_payload_suggest_v2/metadata.yaml:10]:
```suggestion
  schedule: daily
  change_controlled: true
```

File: sql/moz-fx-data-shared-prod/contextual_services_derived/request_payload_tiles_v2/metadata.yaml [hunk 2]
```diff
@@ -4,6 +4,7 @@ description: |-
 owners:
   - skahmann@mozilla.com
   - akommasani@mozilla.com
+  - mozilla/request_payload_reviewers
 labels:
   incremental: true
   schedule: daily
```
Comment (lucia-vargas-a) [inline sql/moz-fx-data-shared-prod/contextual_services_derived/request_payload_tiles_v2/metadata.yaml:10]:
```suggestion
  schedule: daily
  change_controlled: true
```

**Review by lucia-vargas-a — CHANGES_REQUESTED** [review]

File: sql/moz-fx-data-shared-prod/contextual_services_derived/request_payload_suggest_v2/metadata.yaml [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
+  The table is labeled as "change_controlled", which implies
+  that changes require the approval of at least one owner.
 owners:
   - skahmann@mozilla.com
   - akommasani@mozilla.com
+  - mozilla/request_payload_reviewers
 labels:
   incremental: true
   schedule: daily
-  owner1: skahmann
```
[the remark this answers is not in the document]
Comment (alekhyamoz) [reply sql/moz-fx-data-shared-prod/contextual_services_derived/request_payload_suggest_v2/metadata.yaml:10]: so the owners listed here i.e. Sydney and I are just dag owners and are not part of the `request_payload_reviewers`. Wil and Rebecca would be the change control decision makers

**Review by lucia-vargas-a — APPROVED** [review]
The change control implementation LGTM! The only question is about the duplicated owners, if it's necessary.

**Pull request merged**